recipe/meta.yaml

```diff
@@ -19,11 +19,14 @@ build:
     - jupyter-troubleshoot = jupyter_core.troubleshoot:main
 
 requirements:
+  build:
+    - python
```

Comment:
Python should not be required as a build dependency since we are not cross compiling/building.